graphene/types/tests/test_base64.py

```diff
@@ -0,0 +1,101 @@
+import base64
+
+from graphql import GraphQLError
+
+from ..objecttype import ObjectType
+from ..scalars import String
+from ..schema import Schema
+from ..base64 import Base64
+
+
+class Query(ObjectType):
+    base64 = Base64(_in=Base64(name="input"), _match=String(name="match"))
+    bytes_as_base64 = Base64()
+    string_as_base64 = Base64()
+    number_as_base64 = Base64()
+
+    def resolve_base64(self, info, _in=None, _match=None):
+        if _match:
+            assert _in == _match
+        return _in
+
+    def resolve_bytes_as_base64(self, info):
+        return b"Hello world"
+
+    def resolve_string_as_base64(self, info):
+        return "Spam and eggs"
+
+    def resolve_number_as_base64(self, info):
+        return 42
+
+
+schema = Schema(query=Query)
+
+
+def test_base64_query():
+    base64_value = base64.b64encode(b"Random string").decode("utf-8")
+    result = schema.execute(
+        """{ base64(input: "%s", match: "Random string") }""" % base64_value
```

Comment:
```suggestion
        """{ base64(input: "{}", match: "Random string") }""".format(base64_value)
```

Reply:
Using `{}` to format this query means escaping other curly brackets -- which is probably the reason the old formatting is used --, but it can be done:

```diff
-        """{ base64(input: "%s", match: "Random string") }""" % base64_value
+        """{{ base64(input: "{}", match: "Random string") }}""".format(base64_value)
```

I've applied the change above.

Reply:
Oh good point. Thanks for making the change